lib/cocoapods/installer/analyzer.rb

```diff
@@ -429,59 +433,33 @@ def generate_target(target_definition, pod_targets)
           end
         end
 
-        target.pod_targets = filter_pod_targets_for_target_definition(pod_targets, target_definition)
+        target.pod_targets = filter_pod_targets_for_target_definition(target_definition, pod_targets)
```

Comment:
swapped the two parameters for symmetry since all other methods use `target_definition` first and `pod_targets` second.